### PR TITLE
Home work5

### PR DESCRIPTION
**Pull request by SerhiiRiabko**
The text file was added as an example for 3 questions of Homework
I believe that in the folder can be added any file for using code

### CODE REVIEW

File: Lesson_5/home-work5_1/home-work5_1.py [hunk 1]
```diff
@@ -0,0 +1,18 @@
+import random
+import os
+import pickle
+
+# my_list = [(random.randint(1, 100), random.randint(1, 100), random.randint(1, 3)) for _ in range(100)] - see this the
+# same action - Is it correct?
```
Comment (bohdandovbsyh) [inline Lesson_5/home-work5_1/home-work5_1.py:6]: yes. It is a list comprehension. We will learn it in the future

File: Lesson_5/home-work5_1/home-work5_1.py [hunk 2]
```diff
@@ -0,0 +1,18 @@
...
+my_list = []
+
+for _ in range(100):
+    tuple_element = (random.randint(1, 100), random.randint(1, 100), random.randint(1, 3))
+    my_list.append(tuple_element)
+
+print(my_list)
+folder = os.chdir('./..')
+os.mkdir('./text_files')
```
Comment (bohdandovbsyh) [inline Lesson_5/home-work5_1/home-work5_1.py:15]: Please read task more carefully. You should have another naming of the dirs

File: Lesson_5/home-work5_3/home-work5_3.py [hunk 3]
```diff
@@ -0,0 +1,15 @@
+import os
+import re
+
+with open('../text_files_for_3d/text.txt') as file:
+    text = file.read()
+    text_s = text.lower()
```
Comment (bohdandovbsyh) [inline Lesson_5/home-work5_3/home-work5_3.py:6]: what does _s mean?

Comment (SerhiiRiabko) [reply Lesson_5/home-work5_3/home-work5_3.py:6]: I read the file, and make the text lowercase text_s = text_small

Comment (SerhiiRiabko) [reply Lesson_5/home-work5_3/home-work5_3.py]: I did changes

**Pull request merged**